evap/evaluation/models.py

```diff
@@ -470,7 +470,10 @@ def evaluation_ends_soon(self):
 
     @property
     def days_until_evaluation(self):
-        return (self.vote_start_datetime.date() - date.today()).days
+        days_left = (self.vote_start_datetime.date() - date.today()).days
+        if days_left == 0 and self.vote_start_datetime < datetime.now():
+            return -1
+        return days_left
```

Comment:
the `days_left == 0` doesn't change the output, right? if yes, i would propose

```
if self.vote_start_datetime < datetime.now():
    return -1
return (self.vote_start_datetime.date() - date.today()).days
```

Reply:
this would always return -1 if the evaluation start was in the past. currently, courses where the start was earlier are sorted on top of the staff course list. we need to know the actual number here.

Reply:
right.

maybe 
```
days_left = (self.vote_start_datetime.date() - date.today()).days
if self.vote_start_datetime < datetime.now():
    days_left -= 1
return days_left
```
this makes this less special-case-y, doesn't have the weirdness that an evaluation start today and yesterday both result in a -1, which also might make the sorting inconsistent, and avoids the "return -1" which looks a bit like an error code to me.